src/test/resources/allure/allure.properties

```diff
@@ -0,0 +1 @@
+allure.results.directory=target/allure-results
```

Comment:
If you move `allure.properties` from `src/test/resources/allure` directory to `src/test/resources` this configuration will work.

Reply:
Moved allure.properties from src/test/resources/allure directory to src/test/resources